src/Ledger/GovernanceActions.lagda

```diff
@@ -311,8 +305,8 @@ instance
 data
 \end{code}
 
-The relation \verb!_⊢_⇀⦇_,ENACT⦈_! is the transition relation for enacting a governance action.
-It represents how the \AgdaBound{EnactState} changes when a specific governance action is enacted
+The relation \ENACTsyntax is the transition relation for enacting a governance action.
+It represents how the \agdaboundEnactState changes when a specific governance action is enacted
```

Comment:
Why not `\EnactState`?

Reply:
Well, because there are two macros for `EnactState`; I used the name `\EnactState` for `\AgdaRecord{EnactState}`.  Here we need `\AgdaBound{EnactState}`, so I have a separate macro called `\agdaboundEnactState` for that one.